docs/faq/proof-of-stake/general.md

```diff
@@ -0,0 +1,178 @@
+
+Both txfees and ticketfees will not be returned when a ticket votes, expires, or misses a vote.
+
+Returning the funds back to your wallet after a ticket votes or is revoked does not generate a fee.
+
+---
+
+#### **5. What is a stake pool?**
+
+A stake pool is the equivalent of a mining pool, but for the PoS system. Through the options in your Decred wallet, you can give your voting rights to a stake pool. If your ticket is selected to vote,
+the stake pool will cast the vote for you and you are rewarded with the PoS reward minus the stake pool fee (usually 4-5%). Unlike the mining pools, the PoS reward is not split amongst the users of the stake pool, but the full reward goes to the owner of the ticket.
+
+A stake pool will allow you to buy tickets but not have your own wallet unlocked and always online. It is important to note that your funds never leave your wallet. You are not sending anything to the pool,
+just giving it authority to vote on your behalf. A stake pool cannot access your funds. 
+
+Stake pools will usually implement multi-wallet redundancy by having many wallets physically distributed around the globe. This means there's less chance of a vote being missed because one wallet is down.
+It also reduces latency between the wallet and network which can reduce the chance of a vote being missed.
+
+---
+
+#### **6. What happens to my funds when I buy a ticket?**
+
+Funds used to purchase tickets are locked until the ticket is selected to vote or it expires. They are unable to leave your wallet and will appear in the "lockedbytickets" section of your wallet.
+If your ticket is selected to vote, you will be refunded the total amount paid for the ticket and the PoS reward.
+
+The only cost for using this system are the fees you set. The txfee will be charged for the transaction of your ticket across the network. The ticketfee will be charged for incorporation into the
+ticket pool, and is paid to the PoW miners. If your ticket does not get mined, there is no charge as the transaction never made it onto the network.
+
+In case your ticket isn't selected to vote after 40960 blocks (about 4 months), the system will revoke your ticket and the decred you paid for it (minus fees) will be transferred back into your wallet. 
+
+---
+
+#### **7. What are the chances of my ticket voting?**
+
+The PoS system in Decred uses a Poisson distribution to determine the chances of a ticket voting at any given time. Given the target pool size of 40960 tickets, any one ticket has a 50% chance to vote
+within 28 days and a 99.5% chance to vote before expiry. Note that these values will change with the pool size.
+
+
+---
+
+#### **8. What is Proof-of-Stake voting?**
+
+Because the block validation performed by the PoS system behaves like a voting system, it can be used to vote on other issues too.
```

Comment:
A little confusing as to what 'other issues' is in relation to.  Can you clarify.